src/js/services/RecipeService.js

```diff
@@ -58,11 +58,22 @@ function RecipeService($http, $q, AuthService) {
         },
         getSearch: (params) => {
             const {searchTerm, limit = 28, offset = 0} = params;
+            const token = AuthService.getApiToken();
 
-            return parseRecipeArray($http.get(`http://api.kitchen.support/recipes/search/${searchTerm}?limit=${limit}&offset=${offset}`));
+            if (token) {
+                return parseRecipeArray($http.get(`http://api.kitchen.support/recipes/search/${searchTerm}?limit=${limit}&offset=${offset}&api_token=${token}`));
```

Comment:
fwiw, this can be a one-liner using the fact that any valid expression can be in `${}`. Though, for what its worth, query params should probably be handled by the library rather than manually? meh, not a big deal, but just something for future reference